mfr/server/static/js/mfr.js

```diff
@@ -80,6 +80,8 @@
             self.pymParent.iframe.setAttribute('allowfullscreen', '');
             self.pymParent.iframe.setAttribute('webkitallowfullscreen', '');
             self.pymParent.iframe.setAttribute('scrolling', 'yes');
+            self.pymParent.iframe.setAttribute('horizontalscrolling', 'yes');
```

Comment:
1. The `scrolling` attribute for `iframe` is not supported by HTML5, can this be the cause? It seems IE11 dropped several support for `html4` to better support `html5` and `css`: https://msdn.microsoft.com/en-us/library/bg182625(v=vs.85).aspx.
2. Is `horizontalscrolling` and `verticalscrolling` a valid `iframe` attributes? I can't find it though.